keadm/cmd/keadm/app/cmd/edge/image.go

```diff
@@ -41,13 +41,10 @@ func request(opt *common.JoinOptions, step *common.Step) error {
 	}
 
 	step.Printf("Copy resources from the image to the management directory")
-	dirs := map[string]string{
-		util.KubeEdgePath: filepath.Join(util.KubeEdgeTmpPath, "data"),
-	}
 	files := map[string]string{
-		filepath.Join(util.KubeEdgeUsrBinPath, util.KubeEdgeBinaryName): filepath.Join(util.KubeEdgeTmpPath, "bin", util.KubeEdgeBinaryName),
+		filepath.Join(util.KubeEdgeUsrBinPath, util.KubeEdgeBinaryName): filepath.Join(util.KubeEdgeUsrBinPath, util.KubeEdgeBinaryName),
 	}
-	if err := runtime.CopyResources(imageSet.Get(image.EdgeCore), dirs, files); err != nil {
+	if err := runtime.CopyResources(imageSet.Get(image.EdgeCore), files); err != nil {
 		return fmt.Errorf("copy resources failed: %v", err)
 	}
```

Comment:
The dir resource copy was deleted, should we keep it?

Reply:
`dirs` resource mapping moved to `files`?

Reply:
yes

Reply:
for now, we don't copy dir, so the dirs input parameter can be removed totally.

Reply:
I mean, is it necessary to write the original mapping in `dirs` into `files`?
At present, the method has supported both the file path and the dir path in files.
Or maybe the original dir mapping is not needed anymore?

Reply:
oh, i see, I'll remove the dir parameter :)

Reply:
@zc2638 remove dir parameter totally, we can extend copy dir functions once we need it, now we only use to copy files, so not support copy dirs. PTAL